pkg/test/echo/common/metrics.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type EchoMetrics struct {
+	HttpRequests *prometheus.CounterVec
+	GrpcRequests *prometheus.CounterVec
+	TcpRequests  *prometheus.CounterVec
+}
+
+var (
+	Metrics = &EchoMetrics{
```

Comment:
We now use opencensus exporters everywhere in the code.

Reply:
I have changed this to use OC